components/com_kunena/site/template/crypsis/layouts/search/results/default.php

```diff
@@ -10,7 +10,35 @@
  **/
 defined('_JEXEC') or die;
 
-if (empty($this->results)) return;
+if (empty($this->results)) {
+	return;
+}
+$config = KunenaFactory::getConfig();
+// Load FancyBox library if enabled in configuration
+if ($config->lightbox == 1)
+{
+	$template = KunenaTemplate::getInstance();
+	if ( $template->params->get('lightboxColor') == 'white') {
+		$this->addStyleSheet('css/fancybox-white.css');
+	}
+	else  {
+		$this->addStyleSheet('css/fancybox-black.css');
+	}
+	$this->addScript('js/fancybox.js');
+	JFactory::getDocument()->addScriptDeclaration('
```

Comment:
The problem by doing that is that the same code is present in two differents places, it's difficult to maintain